auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -851,15 +851,20 @@
         - name: path
           type: keyword
           description: This is the path associated with a unix socket.
+
     - name: messages
-      type: text
+      type: alias
+      migration: true
+      path: event.original
       description: >
         An ordered list of the raw messages received from the kernel that
         were used to construct this document. This field is present if an error
         occurred processing the data or if `include_raw_message` is set
         in the config.
     - name: warnings
-      type: keyword
+      type: alias
+      migration: true
+      path: error.message
```

Comment:
Not a fan of using `error.message` for debugging output. It's not an actual error message from the source, nor is it a processing error, really, according to the description below.

Reply:
It's a processing error. For example, message would be included if an expected field were missing while processing the events from the kernel. It holds any of the error messages that are returned by go-libaudit while processing the messages.

Reply:
Gotcha, I'm good with that, then.